### PR TITLE
Add exercise 4

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+            return strOutput;
+        }
+
+        static void WriteToFile(string content, string filePath)
```
Comment (voquanghoa) [inline Exercise_4/Program.cs]: Use `File.Write/ReadAllText(...)`

File: Exercise_4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+        static string ReadFromFile(string filePath)
+        {
+            string strOutput = "";
+            try
+            {
+                using (StreamReader sr = new StreamReader(filePath))
+                {
+                    string line;
+                    while ((line = sr.ReadLine()) != null)
```
Comment (voquanghoa) [inline Exercise_4/Program.cs]: Use `StringBuffer` or `StringBuilder` instead

Comment (nguyenvanman) [reply Exercise_4/Program.cs]: Done

File: Exercise_4/Program.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_4/Program.cs]: Done

File: Exercise_4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            var xmlString = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(
+                Encoding.ASCII.GetBytes(ReadFile(jsonInputPath)), new XmlDictionaryReaderQuotas())).ToString();
+            WriteFile(xmlOutputPath, xmlString);
+        }
+
+        static string ReadFile(string filePath)
+        {
+            try
+            {
+                return File.ReadAllText(filePath);
```
Comment (voquanghoa) [inline Exercise_4/Program.cs]: You don't need to introduce a method to do exactly the same thing with a built-in method.

Comment (nguyenvanman) [reply Exercise_4/Program.cs]: Done

**Pull request merged**